### DISCUSSION TITLE
Start/Stop deactivation methods for Renault Clio IV - Ph2 -Dci - EDC gearbox

### DISCUSSION THREAD

**Discussion by Trickykid1978** [cedricp/ddt4all#848] [Announcements]
Hello everybody, 

A complete newbie here ... while waiting for a reliable ELM to arrive (i bought a USB and a WIFI ELM327, both with PIC18F25K80 chips) , i started getting familiar with DDT4ALL in demo mode (i am using DDT4ALL Version 5.5.0 with a 2021 database, loading 3618 ECUs in total) . The first task as soon as i get connected, would be to deactivate the start/stop feature completely and for this reason, i watched tons of videos, read forum threads, etc . So far, i have found the following parameters that can disable start/stop but i am looking for the one that i will finally choose.  Have you tried any of these solutions and which one would you choose just to remain on the safe side of things ?  (the third solution comes from a user in Turkey who has posted a video in youtube explaining why this solution is better as it won't generate a SS signal in the dashboard saying No available etc) 
![option1](https://user-images.githubusercontent.com/127010587/222948034-28963132-0a8d-4014-8bb3-f876f8876e33.jpg)
![option2](https://user-images.githubusercontent.com/127010587/222948037-321a2ae6-363b-47d6-adab-f2907bf3ab79.jpg)
![option3](https://user-images.githubusercontent.com/127010587/222948042-1d5e4d0c-6230-438b-98af-80901fe6bf97.jpg)

**Comment by shrlnm** [comment]
Hi. I'am not an expert in disabling START/STOP function but I may suggest the way. IMHO it's may do a job. 
The X98 platform has an STASTO mtc parameter which is responsible for configuring this function. We may generate default configurations with and without this parameter and compare both sets of configurations.  The default configurations may be generated by the VIN

**Comment by AndyIreland** [comment]
do some people say you can disable SS by just removing the extra cable on the 12v battery under the bonnet 

![image](https://user-images.githubusercontent.com/49172217/222961745-aa0e2f19-7cfa-471d-82ee-fd8a1c8e8a4c.png)


- Im not too sure because our Clio 4 has not got SS at all on ours.
 
If you get SS error light on board you can always extinguish it by coding it out on the cluster 

![image](https://user-images.githubusercontent.com/49172217/222961448-50467229-ce3f-4fb0-ab6d-56b5a688506c.png)

also I dont know what year your ph2 is if over 2019 (but I think it would be called clio5 then) but if it is you will have a security gateway firewall on the OBD2 that you have to unlock with a token from renault before making any adjustments on DDT4ALL

**Comment by shrlnm** [comment]
> do some people say you can disable SS by just removing the extra cable on the 12v battery under the bonnet

I suppose it is a battery current sensor. Removing it you will broke SOH procedure and may damage or get drained the battery one day

**Comment by AndyIreland** [comment]
> > do some people say you can disable SS by just removing the extra cable on the 12v battery under the bonnet
> 
> I suppose it is a battery current sensor. Removing it you will broke SOH procedure and may damage or get drained the battery one day

ah right thanks, what is SOH - state of health?

**Comment by shrlnm** [comment]
@AndyIreland yes It is. The car ECUs have quite complex algorithm of battery state assessment and the battery current sensor is one of main sensors for this. 
So it is quite logical that the car disable START/STOP function when it doesn't know the state of the battery.

**Comment by Trickykid1978** [comment]
> Hi. I'am not an expert in disabling START/STOP function but I may suggest the way. IMHO it's may do a job. The X98 platform has an STASTO mtc parameter which is responsible for configuring this function. We may generate default configurations with and without this parameter and compare both sets of configurations. The default configurations may be generated by the VIN

To be honest, i didn't get what you mean :)  I know that start-stop comes alive provided some conditions are met in the car . For example engine temperature (it won't start-stop when engine is still cold) , so it might be possible to set this temperature setting so high as to render start/stop useless as this temperature might never be met ...this is just a guess ...

**Comment by Trickykid1978** [comment]
> do some people say you can disable SS by just removing the extra cable on the 12v battery under the bonnet
> 
> ![image](https://user-images.githubusercontent.com/49172217/222961745-aa0e2f19-7cfa-471d-82ee-fd8a1c8e8a4c.png)
> 
>     * Im not too sure because our Clio 4 has not got SS at all on ours.
> 
> 
> If you get SS error light on board you can always extinguish it by coding it out on the cluster
> 
> ![image](https://user-images.githubusercontent.com/49172217/222961448-50467229-ce3f-4fb0-ab6d-56b5a688506c.png)
> 
> also I dont know what year your ph2 is if over 2019 (but I think it would be called clio5 then) but if it is you will have a security gateway firewall on the OBD2 that you have to unlock with a token from renault before making any adjustments on DDT4ALL

Hi Andy,  i will be trying to deactivate it through DDT4ALL ... based on data available, i think it will be quite safe and easy to do but i would have felt better if i found someone who did it already and used a specific ecu/parameter to do it.  

My Clio is a X98ph2 - Clio IV Ph2  (this is the model in ddt4all) which is the facelifted Clio iv produced 2016-2019 (it is not firewalled or sth)

**Comment by AndyIreland** [comment]
yeah - just make sure you choose the correct ecu(s) to do it. do a search for particular right ECU's in your car with the magnifying glass in DDT4ALL

**Comment by KarelSvo** [comment]
Simply install a jumper on the hood open sensor connector.
Or is an open bonnet reported in the display?

**Comment by Trickykid1978** [comment]
> Simply install a jumper on the hood open sensor connector. Or is an open bonnet reported in the display?

well, to be honest i would like to get a bit involved with ddt4all as there a few other things i might change if start/stop project goes well ...so, generally disabling start/stop feature is not the main/only issue but disabling it properly **with** ddt4all is :)  . In other renault cars' videos and instructions online, it appears as easy as four clicks (i mean it looks easier than the bonnet sensor that you proposed which is also a correct solution to deactivate ss) . What puzzles me is which of the above three commands  would be better/safer to choose ...

**Comment by Trickykid1978** [comment]
> yeah - just make sure you choose the correct ecu(s) to do it. do a search for particular right ECU's in your car with the magnifying glass in DDT4ALL

i haven't yet understood what green and red ECUs mean ... i mean if the command i want to change is in a ECU listed as red (after searching with the magnifying glass) , should i mess with it or not ?  So far, i have studied almost all ecus in demo mode (complete tree - all screens etc)  but i have zero experience of connected mode as i am waiting for reliable hardware - my mini and old blue ELM327 clone doesn't qualify as such). 

Anyway, after posting this, i found this thread https://github.com/cedricp/ddt4all/issues/728 and from what i understand, my above option No 2 (disabling through UPC-EMM) didn't work for this user with a ph2 clio iv . So for the time being, my two options will be no 1 and no 3 :)

**Comment by AndyIreland** [comment]
in found ecu's if they are in RED then it means that DDT has found a ECU , but its not the one it has expected , so you can make changes but be extra careful with your changing of parameters and take extra caution just in case it bricks the ECU software... I mean thats what I have always taken the red ones to be - but that does not mean to say I have not changed parameters on those red ones , because I have and its been fine with me ... (so far! LOL)

**Comment by shrlnm** [comment]
> To be honest, i didn't get what you mean :)

No problem. Sorry for my bad english. I'll try to explain in other words.
Changing anything thru ddt database (especially such complex  functions as start/stop) is not a right way because this function involve many parameters in many ECUs and you have to know all dependecies for doing right changes.
The DDT database is used by the manufacturer's engineers to create configuration templates, which are then used to configure the ECUs for a specific car (VIN) with an individual set of options. A set of options for a particular car is described by a set of MTC parameters. Based on MTC parameters and configuration templates, configurations are created to configure specific blocks. This procedure is done by REPROG software. 
For example, the start/stop function in MTC terms is controlled by two values (STOSTA = function enabled) and (SSTOST= function disabled)
I took an arbitrary VIN of one of the CLIO IV and generated configuration files with these two values. A different configuration was obtained in three ECUs - TDB, BCM and CLIMAT!!!???))). It is strange but Humidity sensor parameters somehow depends on start/stop )).
The main difference is offcourse in BCM. For the VIN  that I chose for the experiment, the following parameters received different values:

END_CLUTCH_USED_FOR_AUTOSTART_CF
UNDERHOOD_CF
E_B_STOPSTARTINHIBIT_CF
E_B_SOFINHIBITION_CF


In your case it may be different set of parameters and it's why I mentioned your VIN in my first message

**Comment by Trickykid1978** [comment]
> atabase (especially such complex functions as start/stop) is not a right way because this function involve many parameters in many ECUs and you have to know all dependecies for doing right changes. The DDT database is used by the manufacturer's engineers to create configuration templates, which are then used to configure the ECUs for a specific car (VIN) with an individual set of options. A set of options for a particular car is described by a set of MTC parameters. Based on MTC parameters and configuration templates, configurations are created to configure specific blocks. This procedure is done by REPROG software. For example, the start/stop function in MTC terms is controlled by two values (STOSTA = function enabled) and (SSTOST= function disabled) I took an arbitrary VIN of one of the CLIO IV and generated configuration files with these two values. A different configuration was obtained in three ECUs - TDB, BCM and CLIMAT!!!???))). It is strange but Humidity sensor parameters somehow depends on start/stop )). The main difference is offcourse in BCM. For the VIN that I chose for the experiment, the following parameters received different values:
> 
> END_CLUTCH_USED_FOR_AUTOSTART_CF

Hi Shrlnm and many thanks for your input ...  i understand what you mean . It makes sense that start stop is connected with clima as the operation of clima, might delay/postpone or even inhibit the start/stop function (meaning that if you have ambient temperature of 35 degrees and you set clima at 20 degrees, the compressor will be working at 100% thus inhibiting the SS function to operate until desired temperature is reached or almost reached) ... on the other hand, i have seen a lot of users canceling the SS without having (or without reporting) any issues or abnormalities on other systems... anyway my VIN , in case you can source sth more specific for my car,  is VF15RBJ0D57602094 ...also, please do check the above three possible solutions and let me know which one appears more harmless than the others (for some reason, my first choice would be solution no 3 (through Injection ECU) :)

**Comment by shrlnm** [comment]
@Trickykid1978  For your VIN the difference is also in three ECUs

For disabling s/s

Cluster_X98_X87_X82_xHA_X98ph2_X87ph2_v7.4_20160112T110528.xml
    StopStartPresent_CF = 0

BCM_T4_VS_DDT_22_20160531T140853.xml
    END_CLUTCH_USED_FOR_AUTOSTART_CF = 0
    UNDERHOOD_CF = 0
    E_B_STOPSTARTINHIBIT_CF = 1
    E_B_SOFINHIBITION_CF = 1

ClimBoxGen3-v6.3_20161121T160703.xml
    CL_WithHumiditySensor = 1

**Comment by Trickykid1978** [comment]
@shrlnm  

The cluster_X98 etc refers to S/S signals on dashboard. If this is set @ 0 , then no S/S related notifications will be shown at dashboard . If S/S is canceled but this setting is left @ 1 , then the ''S/S not available'' or sth will be reported at dash. 

The UNDERHOOD_CF refers to the ''Open hood'' sensor that disables the S/S anyway as @KarelSvo suggested earlier  (if you drive car with hood open or with this sensor disabled, then car will not allow S/S to operate ...

**Comment by shrlnm** [comment]
@Trickykid1978 I don't know exact behaviour of these parameters (. I only know that these parameters depends on present s/s function

**Comment by Trickykid1978** [comment]
@shrlnm  well it is a matter of dependencies i think ... as an example : if i cancel s/s through  BCM_T4_VS_DDT_22_20160531T140853.xml > E_B_STOPSTARTINHIBIT_CF = **1**  but leaving the UNDERHOOD_CF = **1** , the S/S will be canceled but the hood sensor will remain alive (but useless in regards to S/S function) . If i set UNDERHOOD_CF = 0, then this sensor will be canceled and this sensor might affect other vehicle operations besides S/S  etc etc (so the dependecies will never end)...  on the other hand, what would happen if i leave  E_B_STOPSTARTINHIBIT_CF = **0**  and set UNDERHOOD_CF = **0**  ?  From what i understand, the boot sensor will be canceled and , as a result, S/S will also be canceled ?

**Comment by shrlnm** [comment]
@Trickykid1978 I do not know the exact behavior of all these parameters and their dependencies. But I know that the manufacturer's engineers decided to group these parameters and apply them simultaneously in exactly the same combination as I wrote. They know more than me and I tend to trust them).

**Comment by Trickykid1978** [comment]
@shrlnm  yes , you are right but in any case, S/S can be officially deactivated by a physical, Manufacturer's button in the car (which i don't think that once pressed, three different ECU parameters change at once) ... so it is generally possible, acceptable and safe to drive the car without S/S (you just have to press the button everytime you put the ignition ON) .  It would be interesting to see ,once i get connected, if and which parameters change everytime i press the SS button :)

**Comment by shrlnm** [comment]
@Trickykid1978 Yes it's really interesting. Please share the result of your experiment.

**Comment by KarelSvo** [comment]
> ... so it is generally possible, acceptable and safe to drive the car without S/S (you just have to press the button everytime you put the ignition ON) . 


You can easily reverse the function of the button using a relay.  
[click on this example](https://www-dacianer-de.translate.goog/threads/start-stopp-automatisch-ausschalten-so-gehts.25348/?_x_tr_sl=de&_x_tr_tl=en&_x_tr_hl=de&_x_tr_pto=wapp#post-465362)

**Comment by AndyIreland** [comment]
you could follow this video - its for megane, but just apply same parameters but with your particular Clio 4 ECU's that DDT finds:
https://youtu.be/7-vIlxGQkv0

**Comment by Trickykid1978** [comment]
> you could follow this video - its for megane, but just apply same parameters but with your particular Clio 4 ECU's that DDT finds: https://youtu.be/7-vIlxGQkv0

Hi Andy, this is the above solution No2 (that's the only start/stop related parameter in UPC/EMM Ecu in Clio IV) . In Battery gauge screen, i don't get the same parameters as in the Megane (at least in demo mode) . Check this video if you have the time : https://www.youtube.com/watch?v=Avl1Bi_SCXk&lc=UgzJakhKKojS_4sVkx14AaABAg.9mrtRlnpSJt9msBXqRcSMs   (that's solution No3 :) )

**Comment by WhimP990** [comment]
I am sorry for invaision, but where did you get DDT4ALL Version 5.5.0 with a 2021 database, i cant find it =(
I am not afraid to pay for it too.

**Comment by AndyIreland** [comment]
you can get 5.5.0 with 2019 DB included from here - choose 'all cars exe' 
https://www.drive2.ru/b/498093336985338243/

**Comment by Trickykid1978** [comment]
> I am sorry for invaision, but where did you get DDT4ALL Version 5.5.0 with a 2021 database, i cant find it =( I am not afraid to pay for it too.

Hello, i got mine from the link in this  video https://www.youtube.com/watch?v=K9hL-U6oUik  ... but i guess you will source them elsewhere as well if you search : '' Renault DDT2000 Database 11.2021 '' (you just need the ECU folder transferred in DDT4ALL folder).

**Comment by WhimP990** [comment]
Thank you for responses! I already downloaded the 5.5 from drive2.ru, the car i am working is renault megane 2022 and it failed to recognise any ecu but uch, hope new database will help!

Found ECU [MIU] (no relevant ECU file found) :DIAGVERSION [08] SUPPLIER [LGE_VIETNA] SOFT [SW08.11] VERSION [SW08.11]
Found ECU [ADAS-Sub] (no relevant ECU file found) :DIAGVERSION [04] SUPPLIER [A701] SOFT [R3.11212.3] VERSION [9.00]
Found ECU [Navigation] (no relevant ECU file found) :DIAGVERSION [B1] SUPPLIER [BOSCH] SOFT [I_PRJ_RN_AIVI_21] VERSION [5805_211206]
Found ECU [Cluster/TDB] (no relevant ECU file found) :DIAGVERSION [50] SUPPLIER [10VisteonPalmela-ICE ] SOFT [i60.14] VERSION [2044] ECU uses SDS 1003



and i need to connect to the cluster.

**Comment by AndyIreland** [comment]
with a 2022 Renault car you do realise as well that it will surely have a Security Gateway firewall  that will have to be unlocked with a Renault Service Token as well .. definitely as well if you are looking to change any parameters @WhimP990  - well unless you can bypass the gateway by some way 

https://github.com/cedricp/ddt4all/issues/404

**Comment by AndyIreland** [comment]
> Thank you for responses! I already downloaded the 5.5 from drive2.ru, the car i am working is renault megane 2022 and it failed to recognise any ecu but uch, hope new database will help!
> 
> Found ECU [MIU] (no relevant ECU file found) :DIAGVERSION [08] SUPPLIER [LGE_VIETNA] SOFT [SW08.11] VERSION [SW08.11] Found ECU [ADAS-Sub] (no relevant ECU file found) :DIAGVERSION [04] SUPPLIER [A701] SOFT [R3.11212.3] VERSION [9.00] Found ECU [Navigation] (no relevant ECU file found) :DIAGVERSION [B1] SUPPLIER [BOSCH] SOFT [I_PRJ_RN_AIVI_21] VERSION [5805_211206] Found ECU [Cluster/TDB] (no relevant ECU file found) :DIAGVERSION [50] SUPPLIER [10VisteonPalmela-ICE ] SOFT [i60.14] VERSION [2044] ECU uses SDS 1003
> 
> and i need to connect to the cluster.

if its a larger ECU database you want though I think someone posted up (and I think it was up here on github) 'larger ecu.zip' - but I am damned if I can find it now, typical!

**Comment by WhimP990** [comment]
> with a 2022 Renault car you do realise as well that it will surely have a Security Gateway firewall that will have to be unlocked with a Renault Service Token as well .. definitely as well if you are looking to change any parameters @WhimP990 - well unless you can bypass the gateway by some way
> 
> #404

I unlocked it with autel, it said that it was successful. So now i need to config it, hope the 2021/11 database will be enought.

**Comment by AndyIreland** [comment]
ah right - great!

just done a search - This any good?
https://filecr.com/windows/renault-ddt2000/?id=228499221568

**Comment by KarelSvo** [comment]
> Found ECU [Cluster/TDB] (no relevant ECU file found) :DIAGVERSION [50] SUPPLIER [10VisteonPalmela-ICE ] SOFT [i60.14] VERSION [2044]

Cluster_C1A_RUN2_V2.12
Cluster_C1A_RUN2_V2.12_20211105T165123.xml

The identification line in the file has an bug, so it cannot be matched using automatic search.

AutoIdent DiagVersion="80" Supplier="10VisteonPalmela-ICE" Soft="V60.14" Version="2044"/>

change to

AutoIdent DiagVersion="80" Supplier="10VisteonPalmela-ICE" Soft="i60.14" Version="2044"/>

**Comment by WhimP990** [comment]
Thank you very much, i will try and write here, Thank you!

I've noticed that DiagVersion is not the same, does it metter?

**Comment by KarelSvo** [comment]
> I've noticed that DiagVersion is not the same, does it metter?

hex 50 = decimal 80

**Comment by WhimP990** [comment]
Ok, i am noob, so hope you will forgive me. Maybe you know about the security gateway, i need to unlock it once or every time before i try to config something?

**Comment by KarelSvo** [comment]
Reports from others say, after opening, the gateway remains open until the control devices have gone back to sleep.

**Comment by WhimP990** [comment]
Thank you very much, i am waiting for renolink cable and i will try and let you know!

**Comment by JABICHOCU** [comment]
still trying to de-activate  start-strop on my Renault Captur after trying many of the options here with no luck. I was wondering if anyone tried to do it  by raising the minimum percentage of battery needed for it to work (in my case and others its 80) . I found this on emm battery management options>>  C_pct_soczonetravail_min: 80  Do you think this can be changed to a higher number??

**Comment by Trickykid1978** [comment]
> still trying to de-activate start-strop on my Renault Captur after trying many of the options here with no luck. I was wondering if anyone tried to do it by raising the minimum percentage of battery needed for it to work (in my case and others its 80) . I found this on emm battery management options>> C_pct_soczonetravail_min: 80 Do you think this can be changed to a higher number??

Have you tried Option no3 through Injection ecu ?  It didn't work ? 
![222948042-1d5e4d0c-6230-438b-98af-80901fe6bf97](https://user-images.githubusercontent.com/127010587/229364828-7cea2b67-bf9a-472b-94e0-fb8e55e48ae4.jpg)

**Comment by JABICHOCU** [comment]
I have no such ecu on my car list...these are the ones ddt4all  identified on my car:

Cluster_X98_X87_X82_V2.4_20121001T113250 
T4_BCM_DDT2000_SW11_0_20130128T102714 
EMM_EDISON_DDT2000_SW10_V2_20121220T154401
EMM_EDISON_DDT2000_SW11_1_V1_6_3 
EDC17C42_1701_810_170A_400_V02_20150224T131230  
 EDC17C42_1701_810_170A_400_V02_20150224T131230 
 UPA_Transversal_V2.0_valeo_20121218T161808

**Comment by AndyIreland** [comment]
Cluster is your dashboard
UPA is your parking sensors
I dont know why 2 EMM's showed up like that and with different SW - strange that.
I dont know why EDC is showing up 2 times like that , strange again. and not sure what EDC ECU is to be honest off the top of my head.

**Comment by AndyIreland** [comment]
on my renault clio mk4 stop start inhibit is here on my BCM UCH module and I would presume setting to 'true' - I dont have stop and start feature on my clio at all so I cannot check:
![image](https://user-images.githubusercontent.com/49172217/229504785-601e3a7b-a6b2-461b-ae19-26d1150a612f.png)

**Comment by JABICHOCU** [comment]
I got this list from the "green" modules identified by ddt4all, red ones I discarded so as not to run unnecessary risks. Andy, I have tried to change this last one you posted  but no luck...

**Comment by AndyIreland** [comment]
Yes i cannot see any bms setting or stop start entry on your BCM UCH SW11 at all.

Maybe the only thing you can do is take the stop and start wire off your 12v battery then and extinguish the warning light on your cluster dash with DDT4ALL then?

**Comment by JABICHOCU** [comment]
That means leaving the battery control module completely useless...don´t think it is right, because the battery needs to be charged correctly.

**Comment by AndyIreland** [comment]
yeah possibly, I don't know for certain because my clio mk4 hasn't even got stop/start feature at all on ours so i cannot test. 
Just going by what others have said about removing the (thinner) wire on the 12v battery has disabled it, but that they had the message/light on the dash .... but that can be turned off with DDT4ALL.

is that little wire not just a signal wire for the stop and start system to tell it that the battery is charged enough to activate stop and start? - its not to do with the charging as well is it?
I would have thought the charging up to the battery is done with the large black and red lead to the12v battery because you are talking over 45ah arent you from the alternator, that would be too much to send up that thin wire to the battery. 

is this youtube video any help? - its for a capture but should be the similar on the clio mk4 Ph2 ?

https://youtu.be/3_ymayFxvZc

**Comment by Trickykid1978** [comment]
> That means leaving the battery control module completely useless...don´t think it is right, because the battery needs to be charged correctly.

Maybe it would be better to try the ''hood sensor'' solution  https://www.youtube.com/watch?v=rjeImYk8cdo   I haven't done it but it looks safer than disconnecting the bms cable from battery.

**Comment by shrlnm** [comment]
Hello @Trickykid1978 
> It would be interesting to see ,once i get connected, if and which parameters change everytime i press the SS button :)

Have you tried to compare parameters?

**Comment by JABICHOCU** [comment]
> @Trickykid1978 For your VIN the difference is also in three ECUs
> 
> For disabling s/s
> 
> Cluster_X98_X87_X82_xHA_X98ph2_X87ph2_v7.4_20160112T110528.xml StopStartPresent_CF = 0
> 
> BCM_T4_VS_DDT_22_20160531T140853.xml END_CLUTCH_USED_FOR_AUTOSTART_CF = 0 UNDERHOOD_CF = 0 E_B_STOPSTARTINHIBIT_CF = 1 E_B_SOFINHIBITION_CF = 1
> 
> ClimBoxGen3-v6.3_20161121T160703.xml CL_WithHumiditySensor = 1

Hello shrlnm, can you provide a similar ecu list for Captur x87 , please?

**Comment by shrlnm** [comment]
@JABICHOCU I need a VIN. The set of parameters may depends on it

**Comment by shrlnm** [comment]
@JABICHOCU 
```
###################  SSTOST      #########################
------------------------------   Family :  3  :  INSTRUMENT PANEL
SSCU_PRESENT                                            ;0         

------------------------------   Family :  4  :  UCH
UNDERHOOD_CF                                            ;0         
INFOCLUTCH_DIAG_AUTO_DYN_CF                             ;0         
INFOCLUTCH_DIAG_AUTO_STATIC_CF                          ;0         
END_CLUTCH_USED_FOR_AUTOSTART_CF                        ;0         

------------------------------   Family :  8  :  AIR CONDITIONING
CL_WithHumiditySensor                                   ;1  #### EDITED ####

------------------------------   Family :  20  :  UPC
Nxx_EMM_SAC_CF                                          ;1         
Nxx_HW_FIh_CF 1                                         ;1         
Nxx_HW_FIh_CF 2                                         ;1         
Nxx_HW_FIh_CF 4                                         ;1         
Nxx_HW_FIh_CF 5                                         ;1         
Nxx_HW_FIh_CF 6                                         ;1         
Nxx_HW_FIh_CF 12                                        ;1         
Nxx_StopStart_ESM_FIh_CF 0                              ;1         
Nxx_DiagStopStart_CF 0                                  ;1         
Nxx_DiagStopStart_CF 1                                  ;1         
```

**Comment by JABICHOCU** [comment]
Thank you , and so as to disable S/S what do you suggest,then?

**Comment by shrlnm** [comment]
@JABICHOCU all these parameters  are for turning S/S off

**Comment by AndyIreland** [comment]
well have you got the BCM_T4_VS_DDT_22 bcm uch module in your car @JABICHOCU  - set those parameters like that if you have - if not then try removing the connector off the battery or that lead by the bonnet catch and see what happens

**Comment by JABICHOCU** [comment]
OK...thanks again for your help... Bcm unit I know which one it is but what is  UPC  module, is it the engine??

**Comment by shrlnm** [comment]
@JABICHOCU  these modules are installed in your car

```
#########   Family :  2  :  ABS
ABSESC___X10527998_Version_2.5.xml

#########   Family :  3  :  INSTRUMENT PANEL
Cluster_X98_X87_X82_V2.4_20121001T113250.xml

#########   Family :  4  :  UCH
T4_BCM_DDT2000_SW11_0.xml

#########   Family :  5  :  AIRBAG / PRETENSIONERS
RSAT4_ACU_eng_v11_20120925T103840.xml

#########   Family :  8  :  AIR CONDITIONING
X10_X98_X87_X52ClimBoxGen1_draft_C5_20110913T164836.xml

#########   Family :  18  :  POWER ASSISTED STEERING
X98_EPS_DDT2000_C1_020_b.02_20120329T150729.xml

#########   Family :  20  :  UPC
T4_EMM_DDT2000_SW8.4_V1.xml

#########   Family :  41  :  PARKING PROX CAN
UPA_Transversal_V2.0_valeo_20121218T161808.xml

#########   Family :  76  :  RADIO
ULC_v2.6.xml

#########   Family :  99  :  HANDS-FREE MODULE
T4_HFM_DDT2000_SW4_1_5.xml

```

**Comment by JABICHOCU** [comment]
:) That´s more like it for newbies like me...thanx again!

**Comment by shrlnm** [comment]
It's just for not to guess if ECU scanning was done well ))

**Comment by Trickykid1978** [comment]
> Hello @Trickykid1978
> 
> > It would be interesting to see ,once i get connected, if and which parameters change everytime i press the SS button :)
> 
> Have you tried to compare parameters?

Hi, will do so this week and revert

**Comment by Trickykid1978** [comment]
Hello again, 

Today i finally disabled the start stop system of my Clio IV Ph2 dci / EDC ... it was a nice opportunity as i took car for an oil service at a Renault workshop so i took my laptop with me and i used DDT4ALL right outside their door (so in case anything went wrong, i would have immediate support :) ) . 

Anyway, the correct ECU was the Injection one and the whole path can be found here below ...the nice thing with this method is that you don't get any dashboard error or notice ... the start stop button is completely disabled (it doesn't switch anything or light up anymore when you press it) and the car seems to work just fine ... I also added temperature and time indication on the dashboard (it used to show this info only in Medianav) ... so everything seems ok, many thanks everybody for their input and assistance. 
![startstop](https://user-images.githubusercontent.com/127010587/234357940-ef2404c3-4dd0-4185-bab3-3c9addd173ac.jpg)

**Comment by JABICHOCU** [comment]
Well, I tried this option on the same Ecu ,which doesn´t appear on DDT4ALL as listed for my Captur. But I managed to find it on Ecutweaker...although again, without no success. I click on "write" but an error message appears
![ss](https://user-images.githubusercontent.com/66420341/235345126-1f4adff7-5504-4e09-872d-a63bfcadf018.jpg)

**Comment by Trickykid1978** [comment]
> T4ALL as listed for my Captur. But I managed to find it on Ecutweaker...although again, without no success. I click on "write" but an error message appears

Well, the injection ECU was not appearing in my scanned list of ecus either ... so in ddt4all, i just found it with car model drop down menu (i knew it was sth like EDC17...etc.) . I then loaded it up and noticed that it was a live ecu with data etc ... So i would suggest trying again with ddt4all.

Have you made any other changes with ECUtweaker ? i mean, are you certain it can edit parameters or is it working in reading mode only ?

**Comment by Trickykid1978** [comment]
> Hello again,
> 
> Today i finally disabled the start stop system of my Clio IV Ph2 dci / EDC ... it was a nice opportunity as i took car for an oil service at a Renault workshop so i took my laptop with me and i used DDT4ALL right outside their door (so in case anything went wrong, i would have immediate support :) ) .
> 
> Anyway, the correct ECU was the Injection one and the whole path can be found here below ...the nice thing with this method is that you don't get any dashboard error or notice ... the start stop button is completely disabled (it doesn't switch anything or light up anymore when you press it) and the car seems to work just fine ... I also added temperature and time indication on the dashboard (it used to show this info only in Medianav) ... so everything seems ok, many thanks everybody for their input and assistance. ![startstop](https://user-images.githubusercontent.com/127010587/234357940-ef2404c3-4dd0-4185-bab3-3c9addd173ac.jpg)

One short notice : when SS is disabled through Injection ecu, the light on the SS button remains in the state it was when the canceling command was given. So, if the light on the button was off, it will remain permanently off. If it was ON (SS already disabled though the button) , then it will remain permanently ON after the SS deactivation through ddt4all.

**Comment by JABICHOCU** [comment]
Yes...I´ve already done some other changes (mostly on cluster module) and it worked...but this S-S thing  seems impossible on my 2013 car :(

**Comment by Trickykid1978** [comment]
> Yes...I´ve already done some other changes (mostly on cluster module) and it worked...but this S-S thing seems impossible on my 2013 car :(

Which methods have you already tried ? Also, which is your BCM ecu ?

**Comment by JABICHOCU** [comment]
Here is part of it...https://github.com/cedricp/ddt4all/issues/413

**Comment by JABICHOCU** [comment]
And also here>  https://github.com/cedricp/ddt4all/issues/800

**Comment by Trickykid1978** [comment]
So you have already tried through EMM ?  
![captur](https://user-images.githubusercontent.com/127010587/235487395-24a8378f-2a9e-4709-a185-f08c8979e990.jpg)

**Comment by JABICHOCU** [comment]
Yes, unfortunately it does nothing at all...

**Comment by JABICHOCU** [comment]
> > Hello again,
> > Today i finally disabled the start stop system of my Clio IV Ph2 dci / EDC ... it was a nice opportunity as i took car for an oil service at a Renault workshop so i took my laptop with me and i used DDT4ALL right outside their door (so in case anything went wrong, i would have immediate support :) ) .
> > Anyway, the correct ECU was the Injection one and the whole path can be found here below ...the nice thing with this method is that you don't get any dashboard error or notice ... the start stop button is completely disabled (it doesn't switch anything or light up anymore when you press it) and the car seems to work just fine ... I also added temperature and time indication on the dashboard (it used to show this info only in Medianav) ... so everything seems ok, many thanks everybody for their input and assistance. ![startstop](https://user-images.githubusercontent.com/127010587/234357940-ef2404c3-4dd0-4185-bab3-3c9addd173ac.jpg)
> 
> One short notice : when SS is disabled through Injection ecu, the light on the SS button remains in the state it was when the canceling command was given. So, if the light on the button was off, it will remain permanently off. If it was ON (SS already disabled though the button) , then it will remain permanently ON after the SS deactivation through ddt4all.

Hello again, Trickykid1978...can you give us any updates on this one?? Is your car still running well?? no side effects?? thanx

**Comment by Trickykid1978** [comment]
Yes , car is running great , no issues , no faults ...

**Comment by JABICHOCU** [comment]
OK, and you car is k9k   92 c.v.  or 110 c.v?

**Comment by Trickykid1978** [comment]
To be exact, i have a **Renault Clio IV - 2018 (ph2 model)  - dci - 1.5 - 90hp with EDC gearbox**.

**Comment by JABICHOCU** [comment]
OK, mine is 2013 captur dci 90 hp...it is the same engine, I guess. But it is five years older, maybe that´s the reason my ECU´s don´t behave in the same way...thanx anyway for all your help and posting all your experience here

**Comment by Trickykid1978** [comment]
I think you should first identify the actual ecu injection in your car (maybe scan car with PYREN and see which injection ecu is identified there, i mean the exact xml) . Then you can load up this specific ecu in ddt4all (from the drop down menu, not the magnifying glass) and see if it is actually returning live data from car. This is what i did and despite the risk involved, it worked in my car ... It wouldn't be possible for my start-stop to be deactivated if i only worked with green ecus (or even red for that matter as the injection ecu was not even found in my car when i scanned it with ddt4all)

**Comment by JABICHOCU** [comment]
OK, just tried again and  this injection ecu shows "no data" on my car, couldn´t do anything. So I guess the only option left I have is try some of the parameters of the UPC unit. Does anybody know what these ones stand for?? , especially "**Nxx_EMM_SAC_CF**" and "**Nxx_StopStart_ESM_FIh_CF**"  (I have no idea)



Nxx_EMM_SAC_CF       ??                                   ;1         
Nxx_HW_FIh_CF 1       ??                                  ;1         
Nxx_HW_FIh_CF 2    ??                                     ;1         
Nxx_HW_FIh_CF 4   ??                                     ;1         
Nxx_HW_FIh_CF 5     ??                                    ;1         
Nxx_HW_FIh_CF 6      ??                                   ;1         
Nxx_HW_FIh_CF 12   ??                                      ;1         
Nxx_StopStart_ESM_FIh_CF 0                         ;1         
Nxx_DiagStopStart_CF 0 - diagnosis    
Nxx_DiagStopStart_CF 1 - diagnosis               ;1         

I´d like to try and understand as much as possible for avoiding  risks...thanks for your help

**Comment by Furtif** [comment]
https://github.com/cedricp/ddt4all/issues/836

**Comment by JABICHOCU** [comment]
This is a very interesting topic with lots of info...it would be great if it coud be transferred to the new discussion section

**Reply by cedricp** [reply]
Done

**Reply by Furtif** [reply]
> Done

odd you have converted as annoncement but ok is you the boss :)

**Comment by JABICHOCU** [comment]
By any chance does anybody have DTC 926755  after disabling Stop-start??

**Reply by matzel82** [reply]
This is error that's not configured.... activate and ready. It's Not allowed 🚫

**Reply by matzel82** [reply]
This code says that the SS System have to activate.. !!!!!

**Comment by KarelSvo** [comment]
https://github.com/cedricp/ddt4all/issues/829

**Comment by JABICHOCU** [comment]
Yes, but the question is...why does disabling stop-start  in  _ECM/injection_ unit triggers **no error** and  on the contrary doing it on _USM/EMM_ unit does trigger the **error?**

**Reply by matzel82** [reply]
Because the USM look for the status and configuration. Is the status deactivated...than happened the error.

**Comment by HukuTy** [comment]
Hello, can you suggest me a shutdown for my vin vf1hjd20671281857
